### PR TITLE
add explanation

### PR DESCRIPTION
**Pull request by rachelzhang0922**
#### Which issue(s) this PR fixes:

Fixes #

#### How is this PR tested:
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify)

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: docs/references/words.md [hunk 1]
```diff
@@ -5,6 +5,53 @@ sidebar_position: 4
...
+- Vendor lock-in. We all know it's pretty hard to migrate from one public cloud to another. For example, Google Cloud will shutter its IoT Core service in 2023.
+- High cost. Given the big data & massive scale nature of IoT, running complete solutions on public clouds can be very costly.
 
-:::
+### 🤗EdgeXFoundry
+EdgeXFoundry is a popular open source IoT middleware. It was chartered by Dell in July 2015. It comes with many features after over 7 years of development. Moreover, EdgeXFoundry focuses on edge computing, and has a wide range of application scenarios. However, being a 7-year old product also means it was designed in the pre-Kubernetes era, which leads to the most prominent difference between EdgeXFoundry and Shifu. 
+Unlike EdgeXFoundry, Shifu is a Kubernetes native CRD, and it virtualizes every IoT device as a K8s pod. This endows Shifu with many advantages: 
+- A powerful cloud native eco-system. You can find almost every tool you need in the CNCF cloud native landscape. And being k8s native, Shifu is compatible with every single one of them.
```
Comment (tomqin93) [inline docs/references/words.md:44]: should we capitalize `K` in all K8s?

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@rachelzhang0922 added some comments, please take a look

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/words.md [hunk 2]
```diff
@@ -5,6 +5,53 @@ sidebar_position: 4
 
 # 词汇解释
 
-:::caution正在施工
+## Shifu相关
+- Shifu
+  - 基于[Kubernetes](https://kubernetes.io/docs/reference/glossary/?fundamental=true)的高效物联网设备管理开发框架
+  - 开发者通过使用 Shifu，可以更简单地连接、监视和控制任何物联网设备
+  - Shifu是一个[Kubernetes CRD](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)
```
Comment (BtXin) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/words.md:12]: shifu不是CRD

Comment (rachelzhang0922) [reply i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/words.md:12]: 怎么说更好一点？

Comment (BtXin) [reply i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/words.md:12]: EdgeDevice不才是CRD嘛。。。

File: docs/references/words.md [hunk 3]
```diff
@@ -5,6 +5,53 @@ sidebar_position: 4
 
 # Glossary
 
-:::caution Work in Progress
+## Explanation of Shifu
+- Shifu
+  - An efficient IoT device management and application development framework based on [Kubernetes](https://kubernetes.io/docs/reference/glossary/?fundamental=true)
+  - By using Shifu, developers can connect, monitor and control any IoT device more easily
```
[the remark this answers is not in the document]
Comment (rachelzhang0922) [reply docs/references/words.md:11]: i'd stick to the original statement, it emphasizes the latter part

File: docs/references/words.md [hunk 4]
```diff
@@ -5,6 +5,53 @@ sidebar_position: 4
...
+- shifud: runs on each edgeNode and monitors hardware changes (including connection or disconnection of edgeDevices)
+- shifuController: manages the life cycle of deviceShifu, creating/deleting corresponding deviceShifu instances
+## Tools that will be used
+- Docker: Docker is a software technology providing operating-system-level virtualization also known as containers
+- Kind: A tool for running local Kubernetes clusters using Docker container nodes
+- Kubectl: Command line tool for communicating with a Kubernetes cluster's control plane, using the Kubernetes API
+## Comparison to other tools
+### 😙Cloud Providers' IoT solution offerings (AWS IoT Core, Azure IoT Hub, Alibaba Cloud IoT, etc.)
+Public cloud providers boast complete set of features. If you are looking for an end-to-end solution, you should go to public cloud providers. However, there are a few drawbacks.
+- Vendor lock-in. We all know it's pretty hard to migrate from one public cloud to another. For example, Google Cloud will shutter its IoT Core service in 2023.
```
[the remark this answers is not in the document]
Comment (rachelzhang0922) [reply docs/references/words.md:38]: shutter is correct

File: docs/references/words.md [hunk 5]
```diff
@@ -5,6 +5,53 @@ sidebar_position: 4
...
+- High cost. Given the big data & massive scale nature of IoT, running complete solutions on public clouds can be very costly.
```
[the remark this answers is not in the document]
Comment (rachelzhang0922) [reply docs/references/words.md:39]: I think we should stress the high cost here

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/words.md (hunk 2 above)
Comment (rachelzhang0922) [reply i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/words.md:12]: I need a correct statement I could use

**Review by tomqin93 — APPROVED** [review]
LGTM

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/words.md (hunk 2 above)
Comment (BtXin) [reply i18n/zh-Hans/docusaurus-plugin-content-docs/current/references/words.md:12]: 这CRD定义里：https://github.com/Edgenesis/shifu/blob/main/pkg/k8s/crd/install/shifu_install.yml#L18 
https://github.com/Edgenesis/shifu/blob/main/pkg/k8s/crd/install/shifu_install.yml#L145
俩CRD一个叫edgeDevice，一个叫telemetryService，然而并没有一个叫shifu的crd。。。这里写shifu是CRD太confusing了吧 @saiyan86

**Pull request merged**